src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -97,6 +97,8 @@ public partial class HubConnection : IAsyncDisposable
     /// If this event was triggered from a connection error, the <see cref="Exception"/> that occurred will be passed in as the
     /// sole argument to this handler. If this event was triggered intentionally by either the client or server, then
     /// the argument will be <see langword="null"/>.
+    ///
+    /// The <see cref="Task"/> result is fire-and-forget: the <see cref="HubConnection"/> does not wait for it to complete.
```

Comment:
Not sure we should say "fire-and-forget", that is worrying for UI apps as a background task that throws will crash the process. We call the user provided callback and wrap it in a try catch so we will observe and log any exceptions which prevents crashing.

Something like:
`The <see cref="Task"/> result does not block <see cref="HubConnection"/> operations.`